examples/spin-timer/readme.md

```diff
@@ -2,12 +2,12 @@
 
 To test:
 
-* `cargo build --release`
-* Copy: `cp ./target/release/trigger-timer .`
-* `tar czvf trigger-timer.tar.gz trigger-timer`
-* Update the plugin manifest (`trigger-timer.json`):
-  * Get the SHA: `shasum -a 256 trigger-timer.tar.gz` and copy it into the `sha256` field
-  * Update the URL too, to reflect the directory where the tar file is
-* `spin plugin install --file ./trigger-timer.json --yes`
+- `cargo build --release`
```

Comment:
Is there a reason to change these?  (I don't have an opinion about the change - just surprised to see it in this PR.)

Reply:
updated. good catch. think that was some linter changing things for me.